modules/API/pages/gsql-endpoints.adoc

```diff
@@ -6402,7 +6413,7 @@ Sample Request::
 --
 [source,bash]
 ----
-curl -H 'Content-Type: text/plain' -X PUT 'localhost:14240/gsql/v1/udt/files/' -d '<Token Bank file content>'
+curl -H 'Content-Type: text/plain' -X PUT 'localhost:14240/gsql/v1/udt/files/TokenBank' -d '<Token Bank file content>'
```

Comment:
Fixed a doc error for an existing endpoint.
{fileName} is a required parameter, but it was missing.